packages/blockchain-wallet-v4-frontend/src/data/analytics/types/transactions.ts

```diff
@@ -0,0 +1,12 @@
+export enum Events {
+  TRANSACTIONS_EARN_BUTTON_CLICKED = 'Transactions Earn Button Clicked'
```

Comment:
Adding transaction earn event only as we still have [the legacy ones for sell & buy](https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/6db5e21daa04378b4715ea5eda3871239991ea95/packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/types.ts#L12).

Reply:
from which library from segment you get this? usually we reflect segment structure to this place for easier later maintenance, also once when new types file is about to be added it should contains all existing events :)

Reply:
Thanks for the guidance @milan-bc I'll fix this.

Reply:
This is now fixed, please review @milan-bc